rebalance/rebalance.py

```diff
@@ -470,4 +483,5 @@ def init(options, configuration, plugin):
 plugin.add_option("enough-liquidity", 0, "Above this threshold (in msat), a channel's liquidity is big enough. Will be calculated.", "int")
 plugin.add_option("ideal-ratio", "0.5", "Ideal liquidity ratio for big channels. Will be calculated.", "string")
 plugin.add_option("min-amount", "50000sat", "The minimum rebalancable amount.", "string")
+plugin.add_option("rebalance-stop", False, "It stops the ongoing rebalanceall, set by rebalancestop", "flag")
```

Comment:
Whats the benefit of adding `rebalance-stop` as a documented plugin option? It shows up in `lightning-cli listconfigs` and is not intended for the user to see or access. Why not add it as a member to plugin in line 8?: `plugin.rebalance_stop = False`

Reply:
You're right. By the way, I think the same is true for `cltv-final`, `fee-base`, `fee-per-satoshi`, `enough-liquidity`, and `ideal-ratio`.

`feeratio` and `min-amount` aren't options also, but they are parameters for `rebalanceall`.

I think there should be no option here at all.